did-sdk/src/web/server.ts

```diff
@@ -0,0 +1,62 @@
+import http from 'http'
+import { DIDDocument } from 'did-resolver'
+
+const DOC_PATH = '/.well-known/did.json'
+
+export class DidWebServer {
+  _server: http.Server
+  dids: Map<string, DIDDocument> = new Map()
+  whenReady: Promise<void>
+
+  constructor(public port: number) {
+    this._server = http
+      .createServer((req, res) => this._onRequest(req, res))
+      .listen(port)
+    this.whenReady = new Promise((resolve) => {
+      this._server.on('listening', () => resolve())
+    })
+  }
+
+  _onRequest(req: http.IncomingMessage, res: http.ServerResponse) {
```

Comment:
ah a man of culture i see. writing your own request handler 🕴️

Reply:
Heh indeed